cloudinit/util.py

```diff
@@ -1578,6 +1578,17 @@ def get_cmdline():
     return _get_cmdline()
 
 
+def fips_enabled():
+    fips_proc = "/proc/sys/crypto/fips_enabled"
+    if is_Linux() and os.path.exists(fips_proc):
+        contents = load_file(fips_proc).strip()
+        return True if contents == "1" else False
```

Comment:
We probably don't need to check `is_Linux` first as fallback to `False` is the same in all cases if we don't have a proc file. We could just handle this w/ try/except around the load_file attempt if we wanted.
```suggestion
    try:                                                                        
        contents = load_file(fn)
        return contents.strip() == "1"
    except (IOError, OSError):
        # On systems where the proc entry is not available,
        # we assume FIPS is disabled to retain the old behavior.
        return False
```

Reply:
done.